ee/candi/version_map.yml

```diff
@@ -23,3 +23,8 @@ k8s:
       vcd: v1.2.0
     ccm:
       vcd: 1.4.1
+  '1.30':
+    clusterApi:
+      vcd: v1.3.0
+    ccm:
+      vcd: 1.6.0
```

Comment:
We cannot simply change versions of vcd CCM and CSI

Reply:
I had rechecked supported version and returned the version already in use. 
Next version clusterApi requirement 37.2 VMware Cloud Director API